lib/plugins/aws/package/compile/events/apiGateway/lib/restApi.js

```diff
@@ -74,6 +74,31 @@ module.exports = {
       );
     }
 
+    if (!_.isEmpty(this.serverless.service.provider.apiGateway) &&
+      !_.isUndefined(this.serverless.service.provider.apiGateway.minimumCompressionSize)) {
+      const minimumCompressionSize =
+        this.serverless.service.provider.apiGateway.minimumCompressionSize;
+
+      if (!_.isInteger(minimumCompressionSize)) {
+        const message =
+          `minimumCompressionSize must be an integer. You provided ${minimumCompressionSize}.`;
```

Comment:
This log message will look like `minimumCompressionSize must be an integer. You provided 5` if `minimumCompressionSize='5'`. So maybe since JS doesn't have a `repr()` like python.. I think best alternative is `JSON.stringify` to make it clearer what the user provided. That would print: `minimumCompressionSize must be an integer. You provided "5"` which is clearer to the user.
```suggestion
          `minimumCompressionSize must be an integer. You provided ${JSON.stringify(minimumCompressionSize)}.`;
```

Reply:
Updated, thanks!